doc/lua_api.txt

```diff
@@ -5959,20 +5993,87 @@ object you are working with still exists.
       no effect and returning `nil`.
 * `set_velocity(vel)`
     * `vel` is a vector, e.g. `{x=0.0, y=2.3, z=1.0}`
+    * If the object's velocity is fixed by `lock_velocity()` or `set_speed()`,
+      then only the Y component will be set.
+* `set_velocity_vert(vel_y)`
+    * Sets only the Y component of the velocity vector.
+    * `vel_y` is a number (default: `0.0`)
+* `set_velocity_horz(vel_x, vel_z)`
+    * Sets only the X and Z components of the  velocity vector.
+    * `vel_x` is a number (default: `0.0`)
+    * `vel_z` is a number (default: `0.0`)
```

Comment:
It is very common to want to reset an object's horizontal velocity or vertical acceleration to zero, so providing a convenient shorthand for that purpose I feel outweighs the convention with the other methods. Also these paraticular methods are named uniquely and behave similarly amongst themselves, so that should help to obviate any potential confusion.